src/main/fc/settings.c

```diff
@@ -0,0 +1,688 @@
+/*
```

Comment:
oddly, there was also actually a 32 byte saving in doing that when the ARRAYLEN usages were replaced with a reference to a static variable...